File: 2020/territoriali/multicore/statement/multicore.cpp
#include <string>
#include <iostream>
#include <fstream>
#include <cstdio>

#define MAXN 300

using namespace std;

int main() {
    // se preferisci leggere e scrivere da file
    // ti basta decommentare le seguenti due righe:

    //freopen("input.txt", "r", stdin);
    //freopen("output.txt", "w", stdout);

    int T, t, i;
    cin >> T;

    for (t = 1; t <= T; t++) {
        int N, B;
        int C[MAXN], P[MAXN];
        
        // scrivi in questa variabile la risposta
        int nc;

        cin >> N >> B;
        for (i = 0; i < N; i++)
            cin >> C[i] >> P[i];
        
        // RISOLVI QUI IL PROBLEMA!!

        cout << "Case #" << t << ": " << nc << endl;
    }
}

File: 2020/territoriali/multicore/statement/multicore.pas
var
    T, tt, i, N, B : longint;
    C, P : array[0..300] of longint;
    nc: longint; (* scrivi in queste variabili le risposte *)

begin
    (*
    se preferisci leggere e scrivere da file
    ti basta decommentare le seguenti due righe:
    *)

    (*
    assign(input, 'input.txt'); reset(input);
    assign(output, 'output.txt'); rewrite(output);
    *)

    readln(T);

    for tt := 1 to T do begin
        readln(); (* ignora la riga vuota *)
        readln(N, B);
        for i := 0 to N-1 do
            readln(C[i], P[i]);

        (* RISOLVI QUI IL PROBLEMA!! *)

        writeln('Case #', tt, ': ', nc);
    end;
end.
